openapi.yaml

```diff
@@ -9402,6 +9419,7 @@ components:
     X-Api-Key:
       name: X-Api-Key
       in: header
+      x-isApiKey: true
```

Comment:
Explicitly calling out the "fake" API key header param, so that SDKs can internally handle correct provision of the API Key.